.gitignore

```diff
@@ -15,5 +15,6 @@
 # vendor/
 .idea
 visor
+sentinel-visor
```

Comment:
Is a build still using this name somewhere?

Reply:
I use `go build` a lot when testing